product_docs/docs/lasso/4/describe.mdx

```diff
@@ -4,2237 +4,2092 @@ navTitle: Gathered data details
-May have entries in crontab/cron with sensitive data
+**Security impact:** Low &mdash; 
+Might have entries in `crontab/cron` with sensitive data.
 
 ### debug_sources (`debug_sources`)
 
-Count files under `/usr/src/debug` to detect which applications'
+Count files under `/usr/src/debug` to detect the applications whose
 source code is present in the system and facilitate live debugging.
 
-Report output:
-
-  * file `/linux/debug_sources.data`: Sources for GNU debugger
+**Report output:**
 
-**Depth:** surface
+  * File `/linux/debug_sources.data`: Sources for GNU debugger
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### EFM CLI (`efm_cli`)
 
-Get output of efm cluster-status command.
-
-Report output:
+Get output of `efm cluster-status` command.
 
-  * file `/tools/efm/cli/cluster_status.out`: Output of `efm cluster-status cluster_name` command
+**Report output:**
 
-**Depth:** surface
+  * File `/tools/efm/cli/cluster_status.out`: Output of `efm cluster-status cluster_name` command
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### EFM configuration (`efm_configuration`)
 
-EFM properties and nodes configuration files
+EFM properties and nodes configuration files.
 
-Report output:
+**Report output:**
 
-  * file `/tools/efm/config/efm.nodes`: EFM nodes file
-  * file `/tools/efm/config/efm.properties`: EFM properties file
+  * File `/tools/efm/config/efm.nodes`: EFM nodes file
+  * File `/tools/efm/config/efm.properties`: EFM properties file
 
-**Depth:** surface
+**Depth:** Surface
 
-
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### EFM systemctl (`efm_systemctl`)
 
 When EFM services are detected, collects status and cat of the
-corresponding services. Will check for any service which name starts
-with `edb-efm-`
-
-Report output:
+corresponding services. Checks for any service whose name starts
+with `edb-efm-`.
 
-  * file `/tools/efm/systemd/service_name_cat.data`: Output of 'systemctl cat service_name
-  * file `/tools/efm/systemd/service_name_status.data`: Output of 'systemctl status service_name
+**Report output:**
 
-**Depth:** surface
+  * File `/tools/efm/systemd/service_name_cat.data`: Output of `systemctl cat service_name`
+  * File `/tools/efm/systemd/service_name_status.data`: Output of `systemctl status service_name`
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### etcd CLI (`etcd_cli`)
 
-Gathers the output of some `etcdctl` commands, if `etcdctl` is
-available in the server. The commands are: endpoint status and
-endpoint health.
+Gathers the output of some etcdctl commands, if etcdctl is
```

Comment:
Inconsistent formatting: `etcdctl` is a CLI tool with various commands, which would be invoked via the syntax `etcdctl <command>`, e.g. `etcdctl endpoint status` - most other such commands listed here are formatted as code, so this command should be as well.